tobler/util/util.py

```diff
@@ -12,6 +12,33 @@
 from shapely.geometry import Polygon
 
 
+
+def circumradius(resolution):
+    """Find the circumradius of an h3 hexagon at given resolution.
+
+     Parameters
+    ----------
+    resolution : int
+        h3 grid resolution
+    
+    Returns
+    -------
+    circumradius : float
+        circumradius in meters
+    """
+    try:
+        import h3
+    except ImportError:
+        raise ImportError(
+            "This function requires the `h3` library. "
+            "You can install it with `conda install h3-py` or "
+            "`pip install h3`"
+        )
+
+    cr = h3.edge_length(resolution)
+    return 1000 * cr
```

Comment:
What is the point of getting an edge length if we multiply it by 1000 in the end? Can we use it directly? Having a CRS in meters, buffering by `cr` should be enough, not?

Truth is that we need to deal with CRS because those in feet or degrees need to use a different value but unit conversion seems a cleaner solution that brute-force multiply by 1000.

Reply:
pretty sure that's just a unit conversion because `h3.edge_length` defaults to kilometers but we need to buffer in meters. Alternatively i think it could be

```
return h3.edge_length(resolution, 'm')
```

but i see what you're getting at. This would fail, e.g., for a coordinate system  measured in feet. I think i can add a reasonable check for that

Reply:
If it is in kilometres, that it should be `cr / 1000` to get meters, not multiplication, no?

Reply:
Do you know which units can you pass to `edge_length`? We may be able to pass units from CRS directly. That would resolve it all.

Reply:
nope, `km` and `m` only - https://github.com/uber/h3-py/blob/c3593ecae5796526f9c6dc7dc140c7df2251a7f2/src/h3/_cy/edges.pyx#L71-L74

Reply:
i think only meters and kilometers, but not 100%

Reply:
i was thinking something like this https://github.com/sjsrey/tobler/pull/1